client/blocks/continue-as-user/index.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function ContinueAsUser( { userName, redirectUrl } ) {
+	const translate = useTranslate();
+
+	if ( ! userName || ! redirectUrl ) {
+		return null;
+	}
+
+	const redirectLink = <a href={ redirectUrl }>{ userName }</a>;
```

Comment:
Adding a little gravatar would be nice 🙂

Reply:
Added! Much nicer indeed 🙂 

![image](https://user-images.githubusercontent.com/409615/62872397-4a4c7b80-bd15-11e9-950e-32509d5cab3e.png)